src/open_inwoner/pdc/admin/category.py

```diff
@@ -67,11 +30,39 @@ class CategoryProductInline(OrderedTabularInline):
     extra = 1
 
 
+def zaaktypen_select_schema() -> dict:
+    zaaktypen = ZaakTypeConfig.objects.order_by("catalogus__domein", "omschrijving")
+
+    choices = []
+    for zaaktype in zaaktypen:
+        choices.append(
+            {
+                "value": zaaktype.identificatie,
+                "title": f"{zaaktype.catalogus.domein} - {zaaktype.catalogus.rsin}: {zaaktype.omschrijving}",
+            }
+        )
+
+    choices = sorted(choices, key=lambda item: item["title"])
+
+    schema = {
+        "type": "array",
+        "items": {"type": "string", "choices": choices, "widget": "multiselect"},
+    }
+    return schema
+
+
 class CategoryAdminForm(movenodeform_factory(Category)):
+    zaaktypen = JSONFormField(schema=zaaktypen_select_schema)
```

Comment:
I checked it, the `zaaktypen_select_schema` is triggered when the form renders, so it will be updated if new zaaktypes are created after the webapp started